docs/useIsOffline.md

```diff
@@ -0,0 +1,17 @@
+# OneBlink Apps-react | useIsOffline Hook
+
+## `useIsOffline()`
+
+This function is a react hook for determining whether an application is in an offline state.
```

Comment:
Will need to also document the `IsOfflineContextProvider`. I think it makes sense to have those docs in this file as well. Thoughts?

Reply:
yeh probably makes sense